docs/concepts/credentials/2_claiming.md

```diff
@@ -0,0 +1,83 @@
+---
+id: claiming
+title: Claiming
+---
+There are three actors in the KILT workflow: Claimers, Attesters and Verifiers.
```

Comment:
We are already at the second page, should we maybe put this somewhere earlier in the section?

Reply:
was put into the overview page